package.json

```diff
@@ -1,23 +1,23 @@
 {
   "private": true,
   "devDependencies": {
-    "@babel/core": "^7.10.2",
-    "@babel/preset-env": "^7.10.2",
+    "@babel/core": "^7.12.10",
+    "@babel/preset-env": "^7.12.11",
     "autoprefixer": "9.7.4",
     "del": "5.1.0",
     "gulp": "4.0.2",
     "gulp-all": "1.1.0",
     "gulp-babel": "^8.0.0",
     "gulp-cached": "1.1.1",
-    "gulp-clean-css": "4.2.0",
+    "gulp-clean-css": "^4.3.0",
     "gulp-plumber": "1.2.1",
     "gulp-postcss": "8.0.0",
     "gulp-rename": "2.0.0",
     "gulp-replace": "1.0.0",
-    "gulp-sass": "4.0.2",
+    "gulp-sass": "^4.1.0",
     "gulp-sourcemaps": "2.6.5",
     "gulp-watch": "5.0.1",
-    "node-sass": "4.14.1"
+    "node-sass": "^5.0.0"
```

Comment:
I get 96 of these warnings on build:

> cl : command line warning D9025: overriding '/GR-' with '/GR' [D:\Projects\src\Utilities\Lombiq.Gulp.Extensions\node_modules\node-sass\build\src\libsass.vcxproj] [D:\Projects\src\Utilities\Lombiq.Gulp.Extensions\Lombiq.Gulp.Extensions.csproj]

While this seems like an old error (see https://github.com/sass/node-sass/issues/476 and https://github.com/sass/node-sass/issues/467) it still happens for me on a fresh clone. Don't you see them?

Reply:
Any news @scleaver?